### PR TITLE
currency conversion added

### PR DESCRIPTION
**Pull request by stepanets**

### CODE REVIEW

File: src/Ratio.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * @author Pavel Stepanets <pahhan.ne@gmail.com>
+ * @author Artem Dekhtyar <m@artemd.ru>
+ */
+
+namespace ElegantBro\Money;
+
+use ElegantBro\Interfaces\Numeric;
+use Exception;
+
+interface Ratio
```
Comment (tigokr) [inline src/Ratio.php:14]: maybe CurrencyRatio will be better?

**Review by tigokr — COMMENTED** [review]
in addition i think it would be great to
- add more conversion tests
- add checks for bcmath extension availablity
- modify composer.json about bcmath required

what say you?

File: src/Ratio.php (hunk 1 above)
Comment (stepanets) [reply src/Ratio.php:14]: yes, I thought about name a lot

**Comment by stepanets** [comment]
> 
> 
> in addition i think it would be great to
> 
>     * add more conversion tests
What more case I should test?

>     * modify composer.json about bcmath required
https://github.com/elegant-bro/money/blob/master/composer.json#L18

>     * add checks for bcmath extension availablity
Do you think declaration in `composer.json` is not enough?

File: src/Operations/Converted.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+     * @throws Exception
+     */
+    public function amount(): string
+    {
+        return bcmul(
+            $this->origin->amount(),
+            $this->ratio
+                ->of($this->origin->currency(), $this->target)
+                ->asNumber(),
+            4
```
Comment (anikeevda) [inline src/Operations/Converted.php:52]: I would like to pay attention to hardcoded scale number. In different cases could be important to have more precise value. So I think Money interface must be extended with new method - scale.

**Comment by tigokr** [comment]
nice
and you are good

**Review by tigokr — APPROVED** [review]

**Pull request merged**